vmodules/commands/about.ts

```diff
@@ -15,12 +14,12 @@ const metadata = {
 };
 
 metadata.run = async (m, args, gcfg) => {
-  const bot = memory.client;
+  const bot = memory.client!;
 
   const embed = new djs.MessageEmbed()
     .setColor(bot.cfg.colors.default)
-    .setAuthor(`About`, await bot.managers.assets.getIcon(`info`, bot.cfg.colors.default))
-    .setThumbnail(bot.user.displayAvatarURL({ format: `png`, size: 64 }))
+    .setAuthor(`About`, await bot.managers.assets.getIcon(`info`, (bot.cfg.colors.default as string)))
```

Comment:
`bot.cfg.colors.default as string` -> `ColorResolvable as string` which is valid